AppController/djinn.rb

```diff
@@ -5361,20 +5338,7 @@ def remove_appserver_process(app_id, port, secret)
         "application #{app_name}")
     end
 
-    # Tell the AppController at the login node (which runs HAProxy) that this
-    # AppServer isn't running anymore.
-    if my_node.is_login?
-      remove_appserver_from_haproxy(app, my_node.private_ip, port, @@secret)
-    else
-      acc = AppControllerClient.new(get_login.private_ip, @@secret)
-      begin
-        acc.remove_appserver_from_haproxy(app, my_node.private_ip, port)
-      rescue FailedNodeException
-        Djinn.log_warn("Failed to remove appserver from haproxy for app #{app}")
-      end
-    end
-
-    # And tell the AppDashboard that the AppServer has been killed.
+    # Tell the AppDashboard that the AppServer has been killed.
     delete_instance_from_dashboard(app, "#{my_node.private_ip}:#{port}")
```

Comment:
Should this be done by the appmanagerserver? is there already an ability to talk to the dashboard from the python side?